pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/kafka/clients/consumer/PulsarKafkaConsumer.java

```diff
@@ -159,6 +163,15 @@ private PulsarKafkaConsumer(ConsumerConfig config, Deserializer<K> keyDeserializ
         }
     }
 
+    private SubscriptionInitialPosition getStrategy(final String strategy) {
+        switch(strategy) {
+    	    case "earliest":
+    	        return SubscriptionInitialPosition.Earliest;
+    	    default:
+    		    return SubscriptionInitialPosition.Latest;
```

Comment:
Yeah, these things happen. Oh, some tests were broken by my change. We should fix that before merging.

Reply:
Actually, I discovered a few things. When I looked at the failing tests for integration, ```KafkaApiTest``` failed. But looking in detail, I found that we were using ```KafkaConsumer``` instead of ```PulsarKafkaConsumer```. I don't think the test failures are related to my change. But I'm not too sure. Am I right?

Reply:
PulsarKafkaConsumer was shaded to `KafkaConsumer`. so if there are test failures, it should be related.